scripts/generate_linkerscript.py

```diff
@@ -39,9 +42,13 @@
       for function in symbols:
         name = function['name']
         addresses = function['address']
-        if region in addresses:
+        addr = None
+        if itcm_region in addresses:
```

Comment:
We should put a comment here like "*-ITCM regions are runtime overrides for the normal addresses"